frontend/src/scenes/error-tracking/configuration/alerting/ErrorTrackingAlerting.tsx

```diff
@@ -6,14 +6,7 @@ export function ErrorTrackingAlerting(): JSX.Element {
             logicKey="error-tracking"
             type="internal_destination"
             subTemplateId="error-tracking"
-            filters={{
-                events: [
-                    {
-                        id: `$error_tracking_issue_created`,
-                        type: 'events',
-                    },
-                ],
-            }}
```

Comment:
These were getting applied as `forceFilters` so I added the `getFunctionFilters` function instead